haskell/private/cc_libraries.bzl

```diff
@@ -141,12 +141,15 @@ def link_libraries(libs, args, path_prefix = "", prefix_optl = False):
     Args:
       libs: Sequence of File, libraries to link.
       args: Args or List, append arguments to this object.
-      path_prefix: String, a prefix to apply to search directory arguments. If
-        this is a directory, trailing slashes should be included explicitly (this
-        function will not add them automatically).
+      path_prefix: String, a prefix to apply to search directory arguments. A
+        trailing slash will automatically be added to this argument if provided
+        (you do not need to provide one).
       prefix_optl: Bool, whether to prefix linker flags by -optl
 
     """
+    if path_prefix != "":
+        path_prefix += "/"
+
```

Comment:
`"-optl-L"/"-L" + paths.join(path_prefix, "%s")` might be a better alternative. `paths` is already in scope and `paths.join` already checks if `path_prefix` is empty and additionally avoids `//` by checking if `path_prefix` already ends on `/`.

Reply:
Ah, very nice -- fixed.